### PR TITLE
fixed travis.yml to be valid.

### PR DESCRIPTION
**Pull request by jmcampanini**
fixed the `.travis.yml` file.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ cache: bundler
 install: bundle install
 script:
   - bundle exec jekyll build
-  - bundle exec htmlproof ./_site
+  - bundle exec htmlproof _site
```
Comment (jmcampanini) [inline .travis.yml]: cleaned this up to just be the directory name `_site`

File: .travis.yml [hunk 2]
```diff
@@ -20,7 +20,8 @@ deploy:
     skip_cleanup: true
     on:
       branch: develop
-- provider: s3
+
+  - provider: s3
```
Comment (jmcampanini) [inline .travis.yml]: this was the problem. you needed 2 spaces to have it line up as a block under `deploy`.

File: .travis.yml [hunk 3]
```diff
@@ -28,4 +29,4 @@ deploy:
     local-dir: _site
     skip_cleanup: true
     on:
-      branch: master
\ No newline at end of file
+      branch: master
```
Comment (jmcampanini) [inline .travis.yml]: this just changed because we added a blank line at the end, so technically i added a charcater to this line, the NEW LINE character.

**Comment by necampanini** [comment]
+1 in it goes

**Pull request merged**